azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/notepage.vm

```diff
@@ -36,6 +36,12 @@
 </head>
 <body>
 
+<script type="text/javascript">
+  function enableBanner() {
+    sessionStorage.setItem("dismissBanner", null);
```

Comment:
Should we use "false" rather than `null`, for better expression?

Reply:
updated, i think removeItem would be better to avoid conflict of same key.